### PR TITLE
search panel styling

### PR DESCRIPTION
**Pull request by dev7ch**

### CODE REVIEW

File: modules/admin/src/resources/scss/components/_luyasearch.scss [hunk 1]
```diff
@@ -1,78 +1,104 @@
...
-    background: map-get($grays, 100);;
-    padding: $content-padding-x;
+
+}
+
+.luyasearch-form {
+
+    top: 0;
+    width: 440px;
+    background-color: color(gray);;
```
Comment (nadar) [inline modules/admin/src/resources/scss/components/_luyasearch.scss]: remove ";;"

**Comment by dev7ch** [comment]
-  coz its doenst looks very pretty
- make search input sticky on top

image

**Comment by nadar** [comment]
@dev7ch what does it looks when having results? i just see that white empty container with this very strong dark color on top... maybe the white container should not have 100% height? so you could still klick on the icons behind

**Comment by dev7ch** [comment]
yes the container could be faded in on type.

image

it better then before, i u dont believe me check the code from before ;)

**Comment by dev7ch** [comment]
Yes, better

File: modules/admin/src/resources/scss/components/_luyasearch.scss [hunk 2]
```diff
@@ -1,78 +1,104 @@
...
+.luyasearch-form {
+
+    top: 0;
+    width: 440px;
+    background-color: color(gray);
+    padding: $content-padding-x $content-padding-y $content-padding-x 0;
+    margin-bottom: 0;
+    border-bottom-right-radius: $border-radius;
+    position: fixed;
+    z-index: z-index(toast);
```
Comment (nadar) [inline modules/admin/src/resources/scss/components/_luyasearch.scss]: z-index

Comment (nadar) [inline modules/admin/src/resources/scss/components/_luyasearch.scss]: z-index issue

**Comment by nadar** [comment]
@dev7ch personal i dont like that dark color in the sub navigation. Making the search input sticky is good idea, and maybe remove the 100% height is the white container should only have the height of the number of results, so its possible to click on the submenu behind

File: modules/admin/src/resources/scss/components/_luyasearch.scss [hunk 3]
```diff
@@ -1,78 +1,104 @@
...
+.luyasearch-form {
+
+    top: 0;
+    width: 440px;
+    background-color: color(gray);;
+    padding: $content-padding-x $content-padding-y $content-padding-x 0;
+    margin-bottom: 0;
+    border-bottom-right-radius: $border-radius;
+    position: fixed;
+    z-index: z-index(toast);
```
[the remark this answers is not in the document]
Comment (dev7ch) [reply modules/admin/src/resources/scss/components/_luyasearch.scss]: ok i ll do that

**Comment by dev7ch** [comment]
@nadar the height issue is in my mind, i ll try to keep it :)

about the color, what do u think about a bit lighter grey? ( like new bg from user-online panel)

I think we need a darker color here, espacially if we are going to remove the with bottom.

Do you like the search more how it is now, @luyadev/coredev ?

**Comment by nadar** [comment]
Keep the colors as it is just make the search input sticky and remove the container height. Please don't change colors. I assume also that we will remove new user-online panel and replace with the new submenu from #1467

**Comment by dev7ch** [comment]
image
your advice looks sadly pretty bad

**Comment by dev7ch** [comment]
i ll do it white if you dont would like to follow the consistency of the main navigation

**Comment by dev7ch** [comment]
or the use of box shadows helps a bit:

image

this is how the pr looks like now

File: modules/admin/src/resources/scss/components/_luyasearch.scss (hunk 2 above)
Comment (dev7ch) [reply modules/admin/src/resources/scss/components/_luyasearch.scss]: its changed, search is having own z-index element now

**Pull request merged**